os/system-services/built-in-system-services/index.md

```diff
@@ -74,15 +74,15 @@ This service provides the RancherOS user interface by running `sshd` and `getty`
 
 1. If the `rancher.password=<password>` kernel parameter exists, it sets `<password>` as the password for the `rancher` user.
 
-2. If there are no host SSH keys, it generates host SSH keys and saves them under `rancher.ssh.keys` in [cloud-config]({{site.baseurl}}/os/cloud-config/).
+2. If there are no host SSH keys, it generates host SSH keys and saves them under `rancher.ssh.keys` in [cloud-config]({{site.baseurl}}/os/configuration/#cloud-config).
 
 3. Runs `cloud-init -execute`, which does the following:
 
-   * Updates `.ssh/authorized_keys` in `/home/rancher` and `/home/docker` from [cloud-config]({{site.baseurl}}/os/cloud-config/) and metadata.
-   * Writes files specified by the `write_files` [cloud-config]({{site.baseurl}}/os/cloud-config/) setting.
-   * Resizes the device specified by the `rancher.resize_device` [cloud-config]({{site.baseurl}}/os/cloud-config/) setting.
-   * Mount devices specified in the `mounts` [cloud-config]({{site.baseurl}}/os/cloud-config/) setting.
-   * Set sysctl parameters specified in  the`rancher.sysctl` [cloud-config]({{site.baseurl}}/os/cloud-config/) setting.
+   * Updates `.ssh/authorized_keys` in `/home/rancher` and `/home/docker` from [cloud-config]({{site.baseurl}}/os/configuration/#cloud-config) and metadata.
+   * Writes files specified by the `write_files` [cloud-config]({{site.baseurl}}/os/configuration/#cloud-config) setting.
+   * Resizes the device specified by the `rancher.resize_device` [cloud-config]({{site.baseurl}}/os/configuration/#cloud-config) setting.
+   * Mount devices specified in the `mounts` [cloud-config]({{site.baseurl}}/os/configuration/#cloud-config) setting.
+   * Set sysctl parameters specified in  the`rancher.sysctl` [cloud-config]({{site.baseurl}}/os/configuration/#cloud-config) setting.
 
```

Comment:
Do you think we should link to those specific sections instead of hte general cloud-config section?